x-pack/plugins/security_solution/cypress/e2e/detection_alerts/investigate_in_timeline.cy.ts

```diff
@@ -62,9 +68,39 @@ describe('Alerts timeline', () => {
   });
 
   it('Add an empty property to default timeline', () => {
+    // add condition to make sure the field is empty
+    openAddFilterPopover();
+    fillAddFilterForm({ key: 'file.name', operator: 'does not exist' });
     scrollAlertTableColumnIntoView(ALERT_TABLE_FILE_NAME_HEADER);
     addAlertPropertyToTimeline(ALERT_TABLE_FILE_NAME_VALUES, 0);
     openActiveTimeline();
     cy.get(PROVIDER_BADGE).first().should('have.text', 'NOT file.name exists');
   });
 });
+
+describe('Bulk Investigate in Timeline', () => {
+  before(() => {
+    cleanKibana();
+    login();
+    createCustomRuleEnabled(getNewRule());
+    esArchiverLoad('bulk_process');
```

Comment:
As the file we are using is huge, executing the loading and unloading of it is around one minute. As we are doing this in several tests, we are increasing the test execution time a lot. If it is not possible to reduce it, we should consider putting all the tests that are using it in the same cypress file to load it and unload it just once reducing the time execution.

Reply:
@MadameSheema , it makes sense.. I have consolidated all tests in a single test suite. Please check new updates. Thanks.